integrations/purchase/nordic-api-gateway.js

```diff
@@ -0,0 +1,331 @@
+import { v4 as uuid } from 'uuid';
+import request from 'superagent';
+import {
+  ACTIVITY_TYPE_PURCHASE,
+  PURCHASE_CATEGORY_FOOD_SUPERMARKET,
+  PURCHASE_CATEGORY_STORE_DEPARTMENT,
+  PURCHASE_CATEGORY_TRANSPORTATION_FUEL,
+  PURCHASE_CATEGORY_TRANSPORTATION_RAILROAD,
+  PURCHASE_CATEGORY_TRANSPORTATION_TAXI,
+  PURCHASE_CATEGORY_TRANSPORTATION_AUTOMOTIVE_PARKING,
+  PURCHASE_CATEGORY_TRANSPORTATION_AUTOMOTIVE_PARTS,
+  PURCHASE_CATEGORY_HEALTHCARE_PHARMARCY,
+  PURCHASE_CATEGORY_ENTERTAINMENT_AMUSEMENT_PARKS,
+  PURCHASE_CATEGORY_STORE_GARDEN,
+  PURCHASE_CATEGORY_ENTERTAINMENT_LIQUOR_STORE,
+  PURCHASE_CATEGORY_STORE_PET,
+  PURCHASE_CATEGORY_ENTERTAINMENT_MOVIE_THEATER,
+  PURCHASE_CATEGORY_STORE_BOOKS,
+  PURCHASE_CATEGORY_STORE_BARBER_BEAUTY,
+  PURCHASE_CATEGORY_STORE_ELECTRONIC,
+  PURCHASE_CATEGORY_STORE_HOUSE_FURNISHING,
+  PURCHASE_CATEGORY_STORE_CLOTHING,
+  PURCHASE_CATEGORY_FOOD_RESTAURANT,
+  PURCHASE_CATEGORY_HEALTHCARE_DOCTOR,
+} from '../../definitions';
+import isReactNative from '../utils/isReactNative';
+import { HTTPError, AuthenticationError } from '../utils/errors';
+import { getActivityTypeForCategory, getTransportationModeForCategory } from '../utils/purchases';
+
+import env from '../loadEnv';
+import { convertToEuro } from '../utils/currency/currency';
+/*
+Potential improvements:
+- only refetch items since last fetch.
+- only fetch data from selected accounts
+*/
+
+const callbackUrl = isReactNative
+  ? 'com.tmrow.greenbit://oauth_callback'
```

Comment:
this should maybe be a constant we define in utils (and make sure that the different OAuth managers use that constant as well?)